setup.py

```diff
@@ -25,11 +25,10 @@ def read(fname):
         'Intended Audience :: Developers',
         'License :: OSI Approved :: MIT License',
         'Operating System :: OS Independent',
-        'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 3',
     ],
     extras_require={
-        'django': ['Django>=1.8'],
+        'django': ['Django>2.2'],
```

Comment:
If Django 2.2.0 is supported, isn't it supposed to use `>=` here?

Reply:
@renanivo yes! I fix it.